cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -344,6 +344,41 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
                     }
                     set handled = 1
                 }
+            } elseif (pathStart = "git-command-unstage") {
+                set flags = $get(%request.Data("flags",1),"")
+                if (flags '= "") {
+                    set flaglist = $listfromstring(flags,",")
+                    for i=1:1:$listlength(flaglist) {
+                        set flag = $listget(flaglist,i)
+                        set args($increment(args)) = flag
+                    }
+                }
+                set message = $get(%request.Data("message",1),"")
+                if (message '= "") {
+                    set args($increment(args)) = "-m"
+                    set args($increment(args)) = message
+                }
+                set details = $get(%request.Data("details", 1),"")
+                if (details '= "") {
+                    set args($increment(args)) = "-m"
+                    set args($increment(args)) = details
+                }
+                set command = $get(%request.Data("command",1),"")
+
+                set files = ##class(%Library.DynamicObject).%FromJSON($get(%request.Data("files",1),""))
+                set files = files.files
+
+                set sc = ##class(SourceControl.Git.Utils).RunGitCommandReStage(command,.files,args...)
+
+                set %data = ##class(%Stream.TmpCharacter).%New()
+
+                if $$$ISERR(sc) {
+                    do %data.WriteLine("Error!"_$System.Status.GetErrorText(sc))
+                } else {
+                    do %data.WriteLine("Committed files "_files.%ToJSON()_"!")
```

Comment:
Let's display these in a prettier way (e.g., an indented list of filenames on separate lines)